lib/rubocop/cop/minitest/assert_nil.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This cop checks uses of `assert_equal(nil, ..)`
```

Comment:
The same description as `Description` of config/default.yml is a little more detailed.